### PR TITLE
Add bin icon to delete car in my cars scree & Add option to share app in settings screen

### PR DESCRIPTION
**Pull request by Jdbarad**

### CODE REVIEW

**Comment by Jdbarad** [comment]
https://user-images.githubusercontent.com/93068379/163801757-d18fe064-2c0b-4a1a-8710-bcf0c99e717a.mp4

**Comment by Jdbarad** [comment]
@SebaDro review PR

File: gradle.properties [hunk 1]
```diff
@@ -11,4 +11,5 @@
 sdk.dir=/home/matthes/opt/android/sdk
 org.gradle.jvmargs=-Xmx2048M
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+MAPBOX_DOWNLOADS_TOKEN=sk.eyJ1IjoiamRiYXJhZCIsImEiOiJjbDIzMHU4NjgxbGh4M2NscGQ4ZDN2ZHJ2In0.KZxcRGwzBRraA39k9HaKcQ
```
Comment (SebaDro) [inline gradle.properties:15]: You should never ever publish any sensitive credentials on GitHub. I'd strongly suggest to change your token as soon as possible.

File: org.envirocar.app/src/org/envirocar/app/views/carselection/CarSelectionListAdapter.java [hunk 2]
```diff
@@ -150,37 +156,37 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
+//                            mSelectedButton.setChecked(true);
+//
+//                            // Call the callback in order to react accordingly.
+//                            mCallback.onSelectCar(car);
+//                            break;
+//                        default:
+//                            LOG.warn("No action selected!");
+//                    }
+//                })
+//                .show());
```
Comment (SebaDro) [inline org.envirocar.app/src/org/envirocar/app/views/carselection/CarSelectionListAdapter.java:189]: If you feel that this section is not necessary any more, just delete it and do not comment it out. However, I'd prefer to keep the context menu as an optional function for deleting or selecting a car beside the button based functions.

**Review by SebaDro — CHANGES_REQUESTED** [review]

**Comment by SebaDro** [comment]
Hey @Jdbarad. Thanks for your contributions. Your button based delete function looks good to me. However, you should only provide one new feature per pull request. So just create a new pull request that is based on the develop branch for the bin icon implementation. You could also provide a separate pull request for the sharing feature, but this must be discussed internally, first. So if you do so, it won't be merged in the near future.

In addition, the existing space in the car selection view is very limited for each car row. So placing the bin icon beside the radio button is not a good choice, because in the end there is less space for the car name text. 

See the figures below. If you have cars with longer names, you will get a problem with the new approach.

Current view            |  New view
:-------------------------:|:-------------------------:
 Screenshot_20220624_094947_org envirocar app  | Screenshot_20220624_094735_org envirocar app
 
Please, think about a more user friendly design and follow my review comments. If you do so, I will consider it for merge.

File: gradle.properties (hunk 1 above)
Comment (Jdbarad) [reply gradle.properties:15]: It's Test Token

**Pull request closed without merging**